File: 2 Revalida en C++/2 Secuencias/5.cpp
///calcular el tercer lado de un triangulo
#include <bits/stdc++.h>
using namespace std;
int main()
{
    float b,c;
    float alpha,ang,a;
    cout<<"Ingrese b:";
    cin>>b;
    cout<<"Ingrese c:";
    cin>>c;
    cout<<"Ingrese alpha:";
    cin>>alpha;
    ang=alpha*3.14159/180;
    a=sqrt((b*b)+(c*c)-2*b*c*(cos(ang)));
    cout<<"el resultado es="<<a;
    return 0;
}

File: 1 Revalida en python/2 Secuencias/5.py
#calcular el tercer lado de un triangulo
import math
b=int(input("Ingrese un numero") )
c=int(input("Ingrese un numero") )
alpha=int(input("Ingrese un numero") )
ang=alpha*3.14159/180
a=math.sqrt((b*b)+(c*c)-2*b*c*math.cos(ang))
print a